src/IdentityManager2/Api/Models/UserQueryResultResource.cs

```diff
@@ -30,10 +30,8 @@ public UserQueryResultResource(QueryResult<UserSummary> result, IUrlHelper url,
         }
     }
 
-    public class UserQueryResultResourceData : QueryResult<UserSummary>
+    public class UserQueryResultResourceData : QueryResult<UserResultResource>
     {
-        public new IEnumerable<UserResultResource> Items { get; set; }
```

Comment:
dropping this is needed for System.Text.Json 'coz it is lost with new overrides :-/